internal/repositories/queries.go

```diff
@@ -0,0 +1,262 @@
+package repositories
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/DIMO-Network/telemetry-api/internal/graph/model"
+	"github.com/volatiletech/sqlboiler/v4/drivers"
+	"github.com/volatiletech/sqlboiler/v4/queries"
+	"github.com/volatiletech/sqlboiler/v4/queries/qm"
+)
+
+const (
+	// FloatValueCol is the column name for float values.
+	FloatValueCol = "ValueNumber"
+
+	// StringValueCol is the column name for string values.
+	StringValueCol = "ValueString"
+
+	// IntervalGroup is the column alias for the interval group.
+	IntervalGroup = "group_timestamp"
+)
+
+var sourceTranslations = map[string]string{
+	"macron":   "dimo/integration/2ULfuC8U9dOqRshZBAi0lMM1Rrx",
+	"tesla":    "dimo/integration/26A5Dk3vvvQutjSyF0Jka2DP5lg",
+	"autopi":   "dimo/integration/27qftVRWQYpVDcO5DltO5Ojbjxk",
+	"smartcar": "dimo/integration/22N2xaPOq2WW2gAHBHd0Ikn4Zob",
+}
+
+var dialect = drivers.Dialect{
+	// Clickhouse does not like quotes around identifiers, so we set them to spaces since LQ and RQ are not optional.
```

Comment:
Interesting, given [what they say](https://clickhouse.com/docs/en/sql-reference/syntax#identifiers).

Reply:
It worked when I was using the clickhouse-client but not when using the ch-go lib. However, I didn't try backticks, which may give better results.